### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by funzoneq**
Documentation on how to iPXE boot

### CODE REVIEW

**Comment by janeczku** [comment]
@funzoneq please take a look at issue #321 for the correct kernel arguments for settings the persistent disk and the cloud-config datasource. For the available datasources see here: https://github.com/rancherio/os/blob/3338c4ac63597940bcde7e6005f1cc09287062a2/cmd/cloudinit/cloudinit.go#L378

**Comment by funzoneq** [comment]
thanks @janeczku. I was looking for the correct options. I've updated the PR.

**Comment by imikushin** [comment]
LGTM
/cc @ibuildthecloud

File: docs/running-rancheros/server/ipxe/index.md [hunk 1]
```diff
@@ -6,4 +6,28 @@ layout: default
 ## Booting RancherOS via iPXE
 ----
 
+```
+#!ipxe
+# Boot a persistent RangerOS to RAM
```
Comment (Hellslicer) [inline docs/running-rancheros/server/ipxe/index.md]: typo: RancherOS* ;)

**Comment by ibuildthecloud** [comment]
@funzoneq I tested this out and it worked great, but we have refactored the docs at this point.  Could you put in a PR to https://github.com/rancherio/os/blob/gh-pages/docs/running-rancheros/server/ipxe/index.md with this content?  Thanks so much.  I got RancherOS running on vultr w/ iPXE using this.

**Comment by funzoneq** [comment]
@ibuildthecloud Done: https://github.com/rancherio/os/pull/426

**Pull request closed without merging**